turbomoleio/testfiles/utils.py

```diff
@@ -32,15 +32,38 @@
 import numbers
 import numpy as np
 from contextlib import contextmanager
-from monty.os import cd
+from monty.os import cd, makedirs_p
 from monty.json import MSONable, MontyDecoder
 from monty.serialization import dumpfn, loadfn
 from turbomoleio.input.define import DefineRunner
+from turbomoleio.input.utils import get_define_template
 from turbomoleio.output.states import EigerRunner, States
+from turbomoleio.output.files import JobexOutput, EscfOnlyOutput
 from turbomoleio.core.control import Control
+from turbomoleio.core.control import adg, cdg
+from turbomoleio.output.files import exec_to_out_obj
+from turbomoleio.output.parser import Parser
 
 
 TESTDIR = os.path.split(__file__)[0]
+TM_VERSIONS = ['TM_v7.3', 'TM_v7.3.1']
+OUTPUTS_BASENAMES = {'aoforce': ['aceton_full', 'h2_numforce'],
```

Comment:
good point indeed

Reply:
wouldn't it be easier like this:
{'aoforce': {'aceton_full': TM_VERSIONS, 'h2_numforce': ['TM_v7.3', 'TM_v7.3.1']}, ...

so that for each test, we can either specify to test all the versions, or just a specific set of versions. What do you think ?
Another option could be to have an automatic "discovery" of which tests are available for which version, but I'd rather be explicit somehow.

Reply:
I don't see particular advantages for one option or the other. You can probably see best what is more convenient to retrieve the correct reference at runtime. I would also keep control on these. 

Updating this dictionary should probably be included in a list of things to do when generating tests for a new version.

Reply:
Another option could be to put that in the test.yaml file of the generation folder. We could say that by default, the test is supposed to be available for all versions, and if you specify a list of turbomole versions in the test.yaml file, only this list is used. Of course that means that test.yaml is not *only* for generation but also for the running of the tests. Somehow I would be in favor of this solution in the end so that we do not have too much in the util.py file. What do you think ?

Reply:
We could also rename "generation" to "common" or something else that makes it more "general"

Reply:
Discussion with @gpetretto. One configuration file for each TM_v7.X(.Y) listing all the tests to be performed for the output parsing for this specific version.

Reply:
done